docs/source/docs/examples/cards.blade.md

```diff
@@ -23,7 +23,7 @@ title: "Cards"
 
 @component('_partials.code-sample', ['class' => 'p-10 flex justify-center'])
 <div class="max-w-sm rounded overflow-hidden shadow-lg">
-    <img class="w-full" src="{{ $page->baseUrl }}/img/card-top.jpg">
+    <img class="w-full" src="{{ $page->baseUrl }}/img/card-top.jpg" alt="Sunset in the mountains">
     <div class="px-6 py-4">
         <div class="font-bold text-xl mb-2">The Coldest Sunset</div>
```

Comment:
I'm not sure if a heading is needed, if so which one to use, [bootstrap](https://getbootstrap.com/docs/4.0/components/card/) and [foundation](https://foundation.zurb.com/sites/docs/card.html) use `<h4>`, others don't add semantic